test/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just tested and I can see that you will have the same result if you delete test/resources/logback.xml and add in logback-test.xml:
+    <logger name="akka" level="ERROR">
 +        <appender-ref ref="STDOUT"/>
 +    </logger>